python/lsst/afw/image/calib.cc

```diff
@@ -19,60 +19,100 @@
  * the GNU General Public License along with this program.  If not, 
  * see <https://www.lsstcorp.org/LegalNotices/>.
  */
+#include <memory>
+#include <vector>
 
 #include <pybind11/pybind11.h>
 #include <pybind11/operators.h>
 #include <pybind11/stl.h>
 
+#include "numpy/arrayobject.h"
+#include "ndarray/pybind11.h"
+#include "ndarray/converter.h"
+#include "ndarray_fwd.h"
+
+#include "lsst/base.h"  // for CONST_PTR
```

Comment:
Perhaps just do that for this file (so we can remove the `base.h` header include). In the rest of the code use whatever is most convenient for copy and paste reasons and we'll run a regexp search / replace on the whole thing before (or after) merge to master.